cmake/reports/CMakeLists.txt

```diff
@@ -40,6 +44,7 @@ if (CONFIG_BUILD_WITH_TFM)
       -o ${CMAKE_BINARY_DIR}
       ${workspace_arg}
       -d ${report_depth}
+      -s ${sort_key}
```

Comment:
why is this only added for the TF-M reports, and not the general Zephyr report ?

Reply:
This was an error on my part, I've addressed this in `8f2ac38`